bears/general/IndentationBear.py

```diff
@@ -0,0 +1,218 @@
+from coalib.bears.LocalBear import LocalBear
+from bears.general.AnnotationBear import AnnotationBear
+from bears.general.AnnotationBear import starts_within_ranges
+from coalib.parsing.StringProcessing.Core import unescaped_search_for
+from coalib.bearlib.languages.LanguageDefinition import LanguageDefinition
+from coalib.results.SourceRange import SourceRange
+from coalib.results.Result import Result, RESULT_SEVERITY
+from coalib.results.AbsolutePosition import AbsolutePosition
+from coalib.results.Diff import Diff
```

Comment:
what happened to ordering them alphabetically?

actually, looking at [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) again:

> Imports should be grouped in the following order:
> - standard library imports
> - related third party imports
> - local application/library specific imports
>
> You should put a blank line between each group of imports. 

so at least please put bear imports at the top :)